modules/nf-neuro/tensor/ratio/meta.yml

```diff
@@ -0,0 +1,51 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "tensor_ratio"
+description: Split the tensor image and compute the ratio of Dxx to Dyy and Dxx to Dzz for ALPS.
```

Comment:
Specify to which **convention** the tensor image must abide to (I assume it's FSL). Something like :

The tensor elements must be ordered following <convention> convention, that is : Dxx, Dxy, Dxz, Dyy, Dyz, Dzz.